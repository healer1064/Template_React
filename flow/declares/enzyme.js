// @flow

declare module 'enzyme' {

  declare var shallow: Function

}
