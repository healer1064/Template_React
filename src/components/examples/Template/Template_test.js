// @flow
/* eslint-disable max-len */

// Enzyme docs:
// http://airbnb.io/enzyme/docs/api/index.html

import React from 'react'
import { shallow } from 'enzyme'
import Template from './Template'


const mockProps = {
}


it('<Template> renders the Template content', () => {

  const wrapper = shallow(
    <Template
      {...mockProps}
    />
  )

  expect(wrapper).toMatchSnapshot()

})
