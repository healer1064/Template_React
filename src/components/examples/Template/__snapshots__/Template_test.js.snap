exports[`test <Template> renders the Template content 1`] = `
<div
  className="template">
  Template
</div>
`;
